dashboard/test/controllers/api_controller_test.rb

```diff
@@ -664,7 +658,7 @@ def make_progress_in_section(script)
     }
     assert_response :success
 
-    assert_equal script, assigns(:script)
+    assert_equal '[]', @response.body
```

Comment:
nit: Does this test even make sense anymore? Perhaps it should be renamed at least, since asserting an empty response body doesn't seem to match the title "should get text responses..."